sonar-scanner-gradle/gradle-kotlin-project/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+plugins {
+    jacoco
+    id("org.sonarqube") version "5.0.0.4638"
```

Comment:
Need to update to 6.2.0.5505: https://docs.sonarsource.com/sonarqube-server/latest/analyzing-source-code/scanners/sonarscanner-for-gradle/